### DISCUSSION TITLE
Feature request

### DISCUSSION THREAD

**Discussion by perryfrimps14** [wojtryb/Shortcut-Composer#59] [Ideas]
I think it would be very nice if we could integrate animation tools in the plugin, either in the form of pie menus  or misc tool

**Comment by wojtryb** [comment]
Could you be more specific? I don't know what animation tools you're talking about.

**Reply by egorpro22** [reply]
now in order to select something in the menu you need hold the button, select and then release. is it fashionable to make the button have to be pressed 1 time instead of holding and clicking on the desired item to select?

**Reply by wojtryb** [reply]
Current solution requires less inputs, and feels faster. I don't plan to implement alternative ways of activating the icons.

**Reply by perryfrimps14** [reply]
Hi, sorry for abadoning the thread for so long. The main thing is with frame insertion and with hold frames. Things like adding multiple hold frames, inserting keyframes as well as inserting keyframes to the left or right, atm require right clicking on a particular frame then selecting the option. I think it would help be a little bit more streamlined if these would just be group into pie menu or something is my recommendation. What is your opinion on this @wojtryb?

**Reply by wojtryb** [reply]
Those actions are possible to trigger with the pie menus.

If you haven't update the plugin in the last 2 weeks, download 1.4.0 and install it over your current version.

**Reply by perryfrimps14** [reply]
Oh ok thanks for the great work man, really appreciate this plugin 😊
